.gitlab-ci.yml

```diff
@@ -62,15 +62,15 @@ cwl_dependency_is_stand_alone:
   stage: linting_and_dependencies
   script:
     - pwd
-    - ${MAIN_PYTHON_PKG} -m virtualenv venv && . venv/bin/activate && make prepare && make develop extras=[cwl]
+    - ${MAIN_PYTHON_PKG} -m virtualenv venv && . venv/bin/activate && make prepare && make develop extras=[cwl, server]
```

Comment:
It looks like this also doesn't work because it contains a space; the stuff after the space [doesn't make it into the `pip install` command](https://ucsc-ci.com/databiosphere/toil/-/jobs/28601#L472) and so the install fails in the test.